docs/source/dataframe.rst

```diff
@@ -42,13 +42,13 @@ The ``divisions`` attribute, analogous to ``chunks`` in ``dask.array`` is
 particularly important.  The values in divisions determine a partitioning of
 left-inclusive / right-exclusive ranges on the index::
 
-    divisions -- (10, 20, 40)
-    ranges    -- (-oo, 10), [10, 20), [20, 40), [40, oo)
+    divisions -- (0, 10, 20, 40, 50)
+    ranges    -- [0, 10), [10, 20), [20, 40), [40, 50)
```

Comment:
Yes.  I'm actually figuring out now if we want all intervals to be closed on right or closed on left (any insight here is welcome.)  Regardless the extrema should always closed.  Will fix in a bit.